specification/web/resource-manager/Microsoft.CertificateRegistration/stable/2024-11-01/AppServiceCertificateOrders.json

```diff
@@ -0,0 +1,1802 @@
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "ReissueCertificateOrderRequest resource specific properties",
+          "type": "object",
+          "properties": {
+            "keySize": {
+              "format": "int32",
+              "description": "Certificate Key Size.",
+              "type": "integer"
+            },
+            "delayExistingRevokeInHours": {
+              "format": "int32",
+              "description": "Delay in hours to revoke existing certificate after the new certificate is issued.",
+              "type": "integer"
+            },
+            "csr": {
+              "description": "Csr to be used for re-key operation.",
+              "type": "string"
+            },
+            "isPrivateKeyExternal": {
+              "description": "Should we change the ASC type (from managed private key to external private key and vice versa).",
+              "type": "boolean"
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "RenewCertificateOrderRequest": {
+      "description": "Class representing certificate renew request.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "RenewCertificateOrderRequest resource specific properties",
+          "type": "object",
+          "properties": {
+            "keySize": {
+              "format": "int32",
+              "description": "Certificate Key Size.",
+              "type": "integer"
+            },
+            "csr": {
+              "description": "Csr to be used for re-key operation.",
+              "type": "string"
+            },
+            "isPrivateKeyExternal": {
+              "description": "Should we change the ASC type (from managed private key to external private key and vice versa).",
+              "type": "boolean"
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "SiteSeal": {
+      "description": "Site seal",
+      "required": [
+        "html"
+      ],
+      "type": "object",
+      "properties": {
+        "html": {
+          "description": "HTML snippet",
+          "type": "string",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "SiteSealRequest": {
+      "description": "Site seal request.",
+      "type": "object",
+      "properties": {
+        "lightTheme": {
+          "description": "If <code>true</code> use the light color theme for site seal; otherwise, use the default color theme.",
+          "type": "boolean"
+        },
+        "locale": {
+          "description": "Locale of site seal.",
+          "type": "string"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "subscriptionIdParameter": {
```

Comment:
>subscriptionIdParameter [](http://example.com/codeflow?start=5&length=23)

Please eliminate this definition and use a $ref to the SubscriptionIdParameter definition in v6 common-types types.json file.